sale_delivery_state/models/sale_order_line.py

```diff
@@ -0,0 +1,27 @@
+# Copyright 2023 Manuel Regidor <manuel.regidor@sygel.es>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    skip_sale_delivery_state = fields.Boolean(
+        string="Skip Delivery State",
+        compute="_compute_skip_sale_delivery_state",
+        store=True,
```

Comment:
why stored?

Reply:
this way you have the value on the moment you create the line. check that the compute does not depend on the company_id.skip_service_sale_delivery_state